dotnet/src/Connectors/Connectors.OpenAIV2/Core/ClientCore.ChatCompletion.cs

```diff
@@ -97,13 +97,13 @@ private record ToolCallingConfig(IList<ChatTool>? Tools, ChatToolChoice Choice,
 
             // Serialization of this struct behaves as an empty object {}, need to cast to string to avoid it.
             { nameof(completions.FinishReason), completions.FinishReason.ToString() },
-            { LogProbabilityInfoMetadataKey, completions.ContentTokenLogProbabilities },
+            { ContentTokenLogProbabilitiesKey, completions.ContentTokenLogProbabilities },
         };
     }
 
     private static Dictionary<string, object?> GetChatCompletionMetadata(StreamingChatCompletionUpdate completionUpdate)
     {
-        return new Dictionary<string, object?>(4)
+        return new Dictionary<string, object?>
```

Comment:
```suggestion
        return new Dictionary<string, object?>(4)
```

Reply:
I intentionally removed the capacity because there were a few times when new metadata was added or removed, but the capacity was not updated.